README.md

```diff
@@ -41,40 +41,41 @@ credentials_id = "{{ id from credentials.json }}"
 credentials_secret = "{{ secret from credentials.json }}"
 ```
 
-Make sure to put your actual base_url and the id and secret from your `credentials.json` file in place of the placeholders above.
+Make sure to put your actual base_url and the id and secret from your `credentials.json` file in place of the placeholders above. Note that for testing against a local ProKnow instance, the `base_url` should be set to `http://localhost:3005` and the environment must be configured to use Auth0 as the identity provider (and not Keycloak or OIDC).
 
 Next, run the tests with the `nox` command from within your virtual environment.
 
-```
-nox
+```sh
+$ nox
```

Comment:
Doesn't prefixing all of these commands with a dollar make it an invalid shell script? In other words, you couldn't use the copy to clipboard button to copy the command and paste as-is into the bash prompt?

![Screen Shot 2023-01-27 at 7 33 52 AM](https://user-images.githubusercontent.com/6369365/215088457-e2195609-b566-4e12-b08c-64122f124792.png)

Reply:
For the record, this was not addressed because Ryan said that it looks nicer to have dollar signs prefixing shell commands (even if it makes the shell command invalid) and that @sgerace uses this pattern in his documentation. Since this repository will hopefully be retired once we have time for documentation and it's mostly a matter of personal preference, we decided to leave this as-is.